ci-operator/config/redhat-developer/rhdh/redhat-developer-rhdh-release-1.3.yaml

```diff
@@ -4,9 +4,7 @@ build_root:
       FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.16
       RUN yum -y install make wget gcc git httpd-tools
 images:
-- context_dir: .ibm/images
-  dockerfile_path: Dockerfile
-  to: showcase-e2e-runner
+  - from: quay.io/rhdh-community/showcase-e2e-runner:latest
```

Comment:
why latest for 1.3? latest generally means "the stable release in maintenance" which today is 1.4. meanwhile :next maps to 1.5, until we create the `release-1.5` branches, and move main to 1.6.

Reply:
@nickboldt @zdrapela  showcase-e2e-runner is only an image to run playwright tests, this is not the RHDH image.